tutorbackup/patches/local-docker-compose-jobs-services

```diff
@@ -0,0 +1,17 @@
+############# Backup worker service
+backup-job:
+  image: {{ BACKUP_DOCKER_IMAGE }}
+  environment:
+    - MYSQL_HOST={{ MYSQL_HOST }}
+    - MYSQL_PORT={{ MYSQL_PORT }}
+    - MYSQL_ROOT_USERNAME={{ MYSQL_ROOT_USERNAME }}
+    - MYSQL_ROOT_PASSWORD={{ MYSQL_ROOT_PASSWORD }}
+    - MONGODB_HOST={{ MONGODB_HOST }}
+    - MONGODB_PORT={{ MONGODB_PORT }}
+  volumes:
+    - ../apps/caddy:/caddy
```

Comment:
This looks like it's using an approach of mounting caddy's volume into the backup container? That'll be interesting in Kubernetes, where the `caddy` PVC uses the RWO [access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) so it can only ever be accessed by one container. Unless, that is, the `backup` pod is forced to run on the same node as the `caddy` pod, presumably via a  [`required` pod-affinity rule](https://docs.openshift.com/container-platform/4.9/nodes/scheduling/nodes-scheduler-pod-affinity.html). Is that the plan?

Reply:
I am not sure if I understand your comment correctly. Here I am mounting the `<tutor config root>/env/apps/caddy` directory on local host so that I can copy over the `Caddyfile` that is in there.

Reply:
Yes, I know. My question is how you are planning to map this to Kubernetes. If you have a plan for making that work, excellent. If you don't, you might need to rethink the backup/restore strategy for Caddy. :)

Reply:
I haven't looked into k8 bits yet. If the way Caddy is set up in k8s forces me to change this strategy, I can do it then. But right now I don't understand the problem good enough to devise a strategy that will work for both local and k8s :)